OpenRA.Mods.Common/Traits/World/LobbyOptionCategory.cs

```diff
@@ -0,0 +1,26 @@
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+    [Desc("Allows to group MapOptions in categories.")]
+    public class LobbyOptionCategoryInfo : TraitInfo
```

Comment:
We'll also need to add `[TraitLocation(SystemActors.World | SystemActors.World)]` once #19239 has been merged.